app/views/head-lice.nunjucks

```diff
@@ -0,0 +1,133 @@
+{% import "_macros/images.nunjucks" as images %}
+
+{% extends '_layouts/content-simple.nunjucks' %}
+
+{% block local_header %}
+  <div class="local-header--intro">
+    <p>Head lice, also called nits, are very common in young children.
+      They don’t have anything to do with dirty hair.</p>
+  </div>
+{% endblock %}
+
+{% block main_content_before %}
+  <div class="page-section page-section__collapse-bottom">
+    <div class="reading-width">
```

Comment:
@Demotive this feels a bit `div` heavy but we should be wrapping all content in `.reading-width`, right?